docs/posts/macos-ventura-privacy-security-updates.md

```diff
@@ -0,0 +1,88 @@
+
+In addition to these, other Apple services like **Home** will reject incoming invitations unless you have previously interacted with the sender.
+
+### Device Changes
+
+Some other device functionality is limited with Lockdown Mode enabled as well. If you have a Mac with Apple Silicon, connected devices/accessories are not allowed to connect unless your Mac is unlocked and explicit approval of the device is given. Configuration profiles can no longer be installed either, and the ability to enroll in a Mobile Device Management (MDM) system is disabled. These are enterprise management features, which are occasionally abused to control or monitor devices.
+
+## Passkeys
+
+**Passkeys** are likely to be the most impactful new feature for the everyday person's security practices. Passkeys are a cross-platform standard supported by Google, Apple, and Microsoft. Based on the FIDO2 standard, passkeys are the first real effort to replace passwords as your primary mode of authentication.
+
+Using a passkey stored on your phone is supported by most browsers in macOS, but only Safari currently allows you to use a passkey stored on your Mac at this time. Passkeys generated on your iPhone or Mac are stored in iCloud Keychain, which is end-to-end encrypted with your phone and/or computer's lock screen password. In Safari, passkeys are replacing the single-device WebAuthn registration, which allowed you to use your computer as a hardware key with Touch ID.
+
+The benefits of passkeys are fairly limited at the moment, until more services support the standard for primary or multi-factor authentication.
+
+## Rapid Security Responses
+
+Alongside iOS 16, Apple is introducing **Rapid Security Responses** (RSR) to macOS Ventura. RSR allows Apple to ship small security fixes on the fly much faster than before.
+
+Previously, releasing even a small security patch meant Apple has to release a huge multi-gigabyte update package. This was required to preserve the system's cryptographic integrity following changes to how the system volume is handled in macOS Big Sur. With RSR, updates are much smaller, meaning patches can be downloaded much quicker and applied more reliably.
+
+RSR patches are applied immediately, however [they are tied](https://support.apple.com/guide/deployment/whats-new-dep950aed53e/1/web/1.0) to the macOS minor version they are released with, meaning you need to be on the latest available macOS update to receive them, they are not a replacement for regular updates.
+
+What [remains to be seen](https://eclecticlight.co/2022/09/22/apple-can-patch-ventura-on-the-fly-rsr-is-coming/) is what security patches will be released via RSR as opposed to standard security updates. More severe bugs including Mach zone memory leaks and bugs which could cause kernel panics are still likely to require a formal macOS update rather than a quick patch.
```

Comment:
```suggestion
What [remains to be seen](https://eclecticlight.co/2022/09/22/apple-can-patch-ventura-on-the-fly-rsr-is-coming/) is which security patches will be released via RSR as opposed to standard security updates. More severe bugs, including Mach zone memory leaks and bugs which could cause kernel panics, are still likely to require a formal macOS update rather than a quick patch.
```

Prefers "which" as a determiner, over "what".
Embeds parenthetical information with commas.